--- libogg-1.3.6/CMakeLists.txt.orig	2025-06-16 16:02:12.000000000 +0000
+++ libogg-1.3.6/CMakeLists.txt	2025-06-16 16:02:12.000000000 +0000
@@ -154,6 +154,10 @@
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ogg.pc
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
     )
+
+    install(FILES ${PROJECT_SOURCE_DIR}/ogg.m4
+        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/aclocal
+    )
 endif()
 
 if(INSTALL_DOCS)
