--- libiconv-1.18/lib/iconv.c.orig	2024-12-15 11:53:15.000000000 +0000
+++ libiconv-1.18/lib/iconv.c	2024-12-15 11:53:15.000000000 +0000
@@ -662,7 +662,6 @@
 
 int _libiconv_version = _LIBICONV_VERSION;
 
-#if defined __FreeBSD__ && !defined __gnu_freebsd__
 /* GNU libiconv is the native FreeBSD iconv implementation since 2002.
    It wants to define the symbols 'iconv_open', 'iconv', 'iconv_close'.  */
 #define strong_alias(name, aliasname) _strong_alias(name, aliasname)
@@ -674,4 +673,3 @@
 strong_alias (libiconv_open, iconv_open)
 strong_alias (libiconv, iconv)
 strong_alias (libiconv_close, iconv_close)
-#endif
