--- libwebp-1.6.0/CMakeLists.txt.orig	2025-07-08 00:20:00.000000000 +0000
+++ libwebp-1.6.0/CMakeLists.txt	2025-07-08 00:20:00.000000000 +0000
@@ -57,7 +57,6 @@
   else()
     set(CMAKE_FIND_LIBRARY_SUFFIXES .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
   endif()
-  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
   # vwebp does not compile on Ubuntu with static libraries so disabling it for
   # now.
   set(WEBP_BUILD_VWEBP OFF)
@@ -145,6 +144,16 @@
   add_definitions(-DWEBP_DLL)
 endif()
 
+# Compatibility with autoconf configure script's way of setting this
+if(CMAKE_C_BYTE_ORDER STREQUAL "BIG_ENDIAN")
+  set(WORDS_BIGENDIAN TRUE)
+elseif(CMAKE_C_BYTE_ORDER STREQUAL "LITTLE_ENDIAN")
+  set(WORDS_BIGENDIAN FALSE)
+else()
+  set(WORDS_BIGENDIAN FALSE)
+  message(WARNING "Endianness could not be determined.")
+endif()
+
 # pkg-config variables used by *.pc.in.
 set(prefix ${CMAKE_INSTALL_PREFIX})
 set(exec_prefix "\${prefix}")
@@ -423,10 +432,6 @@
 ${CMAKE_CURRENT_SOURCE_DIR}/src/webp/encode.h;\
 ${CMAKE_CURRENT_SOURCE_DIR}/src/webp/types.h")
 
-# Make sure the OBJECT libraries are built with position independent code (it is
-# not ON by default).
-set_target_properties(webpdecode webpdspdecode webputilsdecode webpencode
-                      webpdsp webputils PROPERTIES POSITION_INDEPENDENT_CODE ON)
 configure_pkg_config("src/libwebp.pc")
 
 # Build the webp demux library.
@@ -795,7 +800,7 @@
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
-set(ConfigPackageLocation ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake/)
+set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 install(EXPORT ${PROJECT_NAME}Targets NAMESPACE ${PROJECT_NAME}::
         DESTINATION ${ConfigPackageLocation})
 
--- libwebp-1.6.0/cmake/config.h.in.orig	2025-07-08 00:20:00.000000000 +0000
+++ libwebp-1.6.0/cmake/config.h.in	2025-07-08 00:20:00.000000000 +0000
@@ -108,12 +108,4 @@
 
 /* Define WORDS_BIGENDIAN to 1 if your processor stores words with the most
    significant byte first (like Motorola and SPARC, unlike Intel). */
-#if defined AC_APPLE_UNIVERSAL_BUILD
-# if defined __BIG_ENDIAN__
-#  define WORDS_BIGENDIAN 1
-# endif
-#else
-# ifndef WORDS_BIGENDIAN
-#  undef WORDS_BIGENDIAN
-# endif
-#endif
+#cmakedefine WORDS_BIGENDIAN 1
