--- xz-5.8.2/src/common/mythread.h.orig	2025-12-17 11:36:42.000000000 +0000
+++ xz-5.8.2/src/common/mythread.h	2025-12-17 11:36:42.000000000 +0000
@@ -165,7 +165,7 @@
 mythread_sigmask(int how, const sigset_t *restrict set,
 		sigset_t *restrict oset)
 {
-#if defined(__VMS) || defined(__MINGW32__)
+#if defined(__VMS) || defined(__MINGW32__) || defined(__DEVKITPRO__)
 	(void)how;
 	(void)set;
 	(void)oset;
